ocs_ci/templates/workloads/smallfile/SmallFile.yaml

```diff
@@ -8,12 +8,14 @@ metadata:
   name: smallfile-benchmark
   namespace: my-ripsaw
 spec:
-  test_user: homer_simpson
-  clustername: aws-dec26-2019
+  test_user: avi-liani
```

Comment:
are these changes intentional?

Reply:
no particular reason for the change, will change it back.

Reply:
changed back.